arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -2121,6 +2121,23 @@ void RocksDBEngine::getStatistics(VPackBuilder& builder) const {
     }
   };
 
+  // get string property from each column family and return sum;
+  auto addStrAllCf = [&](std::string const& s) {
```

Comment:
Wouldn't this be `addIntAllCf`?

Reply:
Open to changing.  I was focused at the time on what the rocksdb GetProperty call was using as a return type.  All ColumnFamily calls return std::string.  Yes, I convert its returned strings to int.  Mostly upset that anyone thought returning a number as a string was the correct action.

Reply:
Agreed.